nautobot/core/templatetags/helpers.py

```diff
@@ -616,6 +616,48 @@ def render_uptime(seconds):
     )
 
 
+@library.filter()
+@register.filter()
+def hyperlinked_field(value, hyperlink):
+    """Render a value as a hyperlink."""
+    if not value:
+        return placeholder(value)
+    return format_html('<a href="{}">{}</a>', value, hyperlink)
+
+
+@library.filter()
+@register.filter()
+def render_m2m(value):
+    """Render sorted by model and app_label ManyToManyField value"""
+    if value.exists() is False:
+        return HTML_NONE
+    sorted_content_types = sorted(value, key=lambda ct: (ct.model, ct.app_label))
+
+    output = "<ul>"
+    for content_type in sorted_content_types:
+        output += f"<li>{content_type}</li>"
+    output += "</ul>"
```

Comment:
For content-types specifically it might make sense to render *all* objects, but for M2M in general we'll want to impose some limit.